### PR TITLE
[3800] Add description of attribute to be used on mouse-over of "Encrypt Marked" button in Configuration

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/3800

### CODE REVIEW

**Comment by jvega190** [comment]
@rart I need to revert common-api that was sent with commit selections

File: static-assets/components/cstudio-admin/mods/admin-configurations.js [hunk 1]
```diff
@@ -534,12 +544,32 @@ function moduleLoaded() {
...
           showErrorDialog(e.message);
         }
 
       });
 
+      // Styles are inline because the dialog its not at admin-config container level.
+      $('#encryptHint').click((e) => {
+        e.preventDefault();
+        CStudioAuthoring.Operations.showSimpleDialog(
+          'error-dialog',
```
Comment (rart) [inline static-assets/components/cstudio-admin/mods/admin-configurations.js:557]: Why error?
Also, use a scoped id with some semantic (e.g. encryptionInfoDialog)

File: static-assets/components/cstudio-admin/mods/admin-configurations.js [hunk 2]
```diff
@@ -534,12 +544,32 @@ function moduleLoaded() {
...
+          CStudioAuthoring.Operations.simpleDialogTypeINFO,
+          formatMessage(adminConfigurationMessages.encryptMarked),
+          '<div style="display: flex;">' +
+          '<i class="fa fa-info-circle" aria-hidden="true" style="font-size: 20px; color: #5cc7fa; margin-right: 10px;"></i>' +
+          `<p style="padding: 0; margin: 0">${formatMessage(adminConfigurationMessages.encryptHint)}</p></div>`,
+          [{
```
Comment (rart) [inline static-assets/components/cstudio-admin/mods/admin-configurations.js:563]: Format properly (buttons param). Though I reckon the ok button is what the show dialog does automatically.

**Review by rart — CHANGES_REQUESTED** [review]
Looks like PRs got mixed (i.e. what’s rte is doing in this pr).
Rollback common API reformat.

**Comment by jvega190** [comment]
Closing PR as there are PR mixes (with rte PR) and common-api needs to be reverted. Will create a clean one

**Pull request closed without merging**